src/extract_sf.c

```diff
@@ -222,8 +223,10 @@ int run(char *in_filename, bool use_fmt, int nevn, int run_no) {
 
                 // Only add points within PLIMITSARR borders and with an acceptable chi2.
                 if ((mean - 2*sigma > PLIMITSARR[ci][0] && mean + 2*sigma < PLIMITSARR[ci][1]) &&
-                        (sf_gaus->GetChisquare() / sf_gaus->GetNDF() < SF_CHI2CONFORMITY))
-                    sf_dotgraph[ci][si]->AddPoint(p + SF_PSTEP/2, mean);
+                    (sf_gaus->GetChisquare() / sf_gaus->GetNDF() < SF_CHI2CONFORMITY)){
+                        // Older root compatibility fix
+                        sf_dotgraph[ci][si]->SetPoint(point_index, p + SF_PSTEP/2, mean); point_index++;
+                    }
                 }
```

Comment:
Indentation and the positioning of brackets here is wrong, please check.

Reply:
Cheked!